src/main/java/com/exactpro/th2/check1/configuration/Check1Configuration.java

```diff
@@ -34,6 +38,14 @@
     @JsonProperty(value="rule-execution-timeout", defaultValue = "5000")
     private long ruleExecutionTimeout = 5000L;
 
+    @JsonProperty(value="decimal-precision", defaultValue = "0.005")
+    private double decimalPrecision = 0.005;
+
+    @JsonProperty(value="time-precision", defaultValue = "PT3S")
+    @JsonDeserialize(using = DurationDeserializer.class)
+    @JsonPropertyDescription("The default time precision value uses java duration format")
+    private Duration timePrecision = Duration.parse("PT3S");
```

Comment:
I think we should use a default value that is less than a second. Like 1 ns or 1 μs